web-api-shared/src/postgres.rs

```diff
@@ -53,8 +53,15 @@ pub struct Config {
     /// If true skips running db migrations on start up.
     skip_migrations: bool,
 
-    /// Number of connections in the pool.
+    /// Minimum number of connections in the pool.
+    /// When the pool is built, this many connections will be automatically spun up.
+    /// This value is clamped internally to not exceed `max_pool_size`.
+    /// Default: 0
```

Comment:
❓ should we document the defaults in the comment? this is likely to be forgotten if we change the defaults. and anyone interested in the defaults can just jump to the `Default` impl.

same question for the `max_pool_size`.

Reply:
removed